docs/explore/insights.md

```diff
@@ -1,86 +1,201 @@
-![Screenshot](../assets/explore/insights/insights-tags-dark.png#only-dark){: style="height:100px"}
+![insight-page](../assets/explore/insights/insights-2-light.png#only-light)
+![insight-page](../assets/explore/insights/insights-2-dark.png#only-dark)
 
-* **Date Range:** Customize the time period for analysis (Week, Month, Quarter, Year).
+## Filtering Controls
 
-![Screenshot](../assets/explore/insights/insights-date-light.png#only-light)
-![Screenshot](../assets/explore/insights/insights-date-dark.png#only-dark)
+Filtering Controls allow you to refine the data displayed on the **Insights** page. You can customize the data view based on **Source Datastores**, **Tags**, **Report Date**, and **Timeframe**, ensuring you focus on the specific information that matters to you.
 
-### Insights Summary
+![filters](../assets/explore/insights/filter-3-light.png#only-light)
+![filters](../assets/explore/insights/filter-3-dark.png#only-dark)
 
+| No | Filter | Description |
+| :---- | :---- | :---- |
+| 1. | Select Source Datastores | Select specific source datastores to focus on their data. |
+| 2. | Tags | Filter data by specific tags to categorize and refine results. |
+| 3. | Report Date  | Set the report date to view data from a particular day. |
+| 4. | Timeframe | Choose a timeframe to view data for a specific  (week, month, quarter, and year) |
 
-* **Monitoring Overview**
-    * Data Under Management 
-    * Overall Quality Score
-    * Total of Datastores
-    * Total of Containers
+## Quality Score
 
-    ![Screenshot](../assets/explore/insights/insights-overview-light.png#only-light){: style="height:300px"}
-    ![Screenshot](../assets/explore/insights/insights-overview-dark.png#only-dark){: style="height:300px"}
+**Quality Score** gives a clear view of your data's overall quality. It shows important measures like [**Completeness**](https://userguide.qualytics.io/glossary/#accuracy:~:text=perform%20a%20comparison-,Completeness,-Required%20fields%20are), [**Conformity**](https://userguide.qualytics.io/glossary/#accuracy:~:text=are%20fully%20populated.-,Conformity,-Alignment%20of%20the), [**Consistency**](https://userguide.qualytics.io/glossary/#completeness:~:text=other%20JDBC%20source-,Consistency,-The%20value%20is), [**Precision**](https://userguide.qualytics.io/glossary/#precision:~:text=data%20between%20systems.-,Precision,-Your%20data%20is), [**Timeliness**](https://userguide.qualytics.io/glossary/#timeliness:~:text=internally%20vetted%20datasets.-,Timeliness,-It%20can%20be), [**Volumetrics**](https://userguide.qualytics.io/glossary/#timeliness:~:text=when%20it%E2%80%99s%20expected.-,Volumetrics,-Data%20has%20the), and [**Accuracy**](https://userguide.qualytics.io/glossary/#:~:text=batch%20of%20data-,Accuracy,-The%20data%20represents), each represented by a percentage. This helps you quickly understand the health of your data, making it easier to identify areas that need improvement.  
```

Comment:
Hi Muze, you are pointing to production environment, you need to change the way we are showing the hyperlinks here too